homework-solutions/js/hw-13/task-1.ts

```diff
@@ -0,0 +1,84 @@
+// 1. Объявите переменные для каждого из следующих типов данных с явным указанием типа после символв "двоеточие":
+//     - Число: переменная num1, значение 42.
+//     - Строка: переменная str, значение "Hello, TypeScript!".
+//     - Булево: переменная isComplete, значение true.
+//     - Массив чисел: переменная numbers, значения [1, 2, 3, 4, 5].
+//     - Массив строк: переменная cities, значения ["Minsk", "Warsaw", "London"].
+//     - Объект: переменная person, объект с полями name: "Alice", age: 30, city: "Minsk".
+
+let num1: number = 42;
```

Comment:
Используем `const`, если значение не переопределяется
```suggestion
const num1: number = 42;
```